docs/docs-content/clusters/data-center/nutanix/create-manage-nutanix-cluster.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_label: "Create and Manage Nutanix Cluster"
+title: "Create and Manage Nutanix Cluster"
+description: "Learn how to create and manage Nutanix clusters in Palette."
+hide_table_of_contents: false
+tags: ["data center", "nutanix"]
+---
+
+Palette supports creating and managing Kubernetes clusters deployed to a Nutanix infrastructure. This section guides you in creating a Kubernetes cluster in a Nutanix cloud managed by Palette.
+
+## Prerequisites
+
+- A Nutanix Prism Central cloud account.
+
+- A Private Cloud Gateway (PCG) deployed.
```

Comment:
Let's link to the PCG page this PR created.

Reply:
added.